tests/dummy/app/templates/usage.hbs

```diff
@@ -277,17 +277,23 @@ triggerOnAlphaNumeric(event) {
 
 ### `Ember.TextField` && `Ember.TextArea`
 
-To prevent `ember-keyboard` from responding to keystrokes while an input/textarea is focused, previous versions of ember-keyboard used an intializers
-to reopen `Ember.TextField` and `Ember.TextArea` and apply the `EKOnInsertMixin` and `EKFirstResponderOnFocusMixin`. This ensures that whenever an
-input is focused, other key responders will not fire. This applies to `input` and `textarea` helpers:
+To prevent `ember-keyboard` from responding to keystrokes while an input/textarea is focused,
+you may use the `on-key` modifier with regular `<input>` and `<textarea>` elements and invoke
+`ekEvent.stopPropagation()` and/or `ekEvent.stopImmediatePropagation()`.
+This ensures that whenever an input is focused, other key responders will not fire.
 
 ```htmlbars
-{{input}}
+<input {{on-key '/' this.onSlash}} />
```

Comment:
@lukemelia do you mind to take a look into this example? I'm not 100% sure this is the exactly right way

Reply:
This looks as I would expect.